aggregations-and-analytics/10/10_moving_average.md

```diff
@@ -0,0 +1,41 @@
+# 01 Moving average
```

Comment:
Wouldn't you implement a Moving Average with an `OVER` window usually (covered in "Rolling Aggregations on Times Series Data"? "Overlapping Time Windows" instead? Probably, there are better options.

Reply:
"# 01"needs to be updated to actual number.

Reply:
What about Fixed Interval Windows, since a HOP always emits, almost like a scheduled task?

Reply:
Doesn't TUMBLE do the same?

Reply:
Yes true. Jumping Time Windows?

Reply:
I don't know if coining another term in addition to hop and sliding for the same helps here ;)

Reply:
Then let's settle for Hopping Time Windows :D